test/integration/QueryTest.java

```diff
@@ -202,6 +202,9 @@ public void test_query_undefine() throws IOException {
 
                     // check total count
                     assertEquals(15 - 5, tx.logic().rules().toList().size());
+
+                    // a query that used to trigger a rule should not cause an error
+                    List<ConceptMap> answers = tx.query().match(Graql.parseQuery("match $x isa repo-fork;").asMatch()).toList();
```

Comment:
Add a BDD scenario

Reply:
test existed in BDD, but was ignored because reasoning was not enabled.  https://github.com/graknlabs/behaviour/pull/183 enables these and other tests